### PR TITLE
chore: update channel for contact MMI team and CircleCI step

### PR DESCRIPTION
**Pull request by zone-live**
## **Description**

Updates contact channel to be: `#contact-mmi-team` and the MMI CircleCI e2e step.

## **Related issues**

Fixes:

## **Manual testing steps**

1. Go to this page...
2.
3.

## **Pre-merge author checklist**

- [ ] I've followed [MetaMask Contributor Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask Extension Coding Standards](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've completed the PR template to the best of my ability
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

File: playwright.config.ts [hunk 1]
```diff
@@ -44,7 +43,7 @@ const config: PlaywrightTestConfig = {
     trace: 'on',
     video: 'off',
     /* Run tests headless in local */
-    headless: isHeadless('PLAYWRIGHT'),
+    headless: process.env.HEADLESS === 'true',
```
[the remark this answers is not in the document]
Comment (zone-live) [reply playwright.config.ts:46]: It will fail to run locally because it can't find the `isHeadless`.

[the remark this answers is not in the document]
Comment (zone-live) [reply playwright.config.ts:46]: Screenshot 2024-06-18 at 12 02 31

[the remark this answers is not in the document]
Comment (zone-live) [reply playwright.config.ts:46]: Got it, let's try it out, I've approved https://github.com/MetaMask/metamask-extension/pull/25388 👍🏼

[the remark this answers is not in the document]
Comment (zone-live) [reply playwright.config.ts]: Re-added the `isHeadless`, it's working locally as well after I added the `env.ts` in the docker file. Thank you 👍🏼

**Review by albertolive — APPROVED** [review]

**Review by danjm — APPROVED** [review]

**Pull request merged**